OpenRA.Mods.Common/AI/States/GroundStates.cs

```diff
@@ -20,6 +20,11 @@ protected virtual bool ShouldFlee(Squad owner)
 		{
 			return base.ShouldFlee(owner, enemies => !AttackOrFleeFuzzy.Default.CanAttack(owner.Units, enemies));
 		}
+
+		protected Actor FindClosestEnemy(Squad owner)
+		{
+			return owner.Bot.FindClosestEnemy(owner.Units.FirstOrDefault().CenterPosition);
```

Comment:
I think it is an artifact from my other experimental stuff involving OnDamage() to squads which make HackyAI code doesn't have time to remove invalid members from the squad. I'll change it to First.